src/lib/keys.js

```diff
@@ -53,4 +53,6 @@ export function allKeys( arg ) {
   return arg ? (arg.constructor === Symbol || typeof arg === 'symbol') : Symbol( 'allKeys' );
 }
 
+export const ALL_KEYS = Symbol('ALL_KEYS')
```

Comment:
Yes, I thought about it. But the purpose of `allKeys` function is kinda blurry in my head, could you explain a bit please that why does it have a param check!